arbnode/node.go

```diff
@@ -576,7 +581,13 @@ func createNodeImpl(
 		}
 	} else if l2Config.ArbitrumChainParams.DataAvailabilityCommittee {
 		return nil, errors.New("a data availability service is required for this chain, but it was not configured")
-	} else if config.EigenDA.Enable {
+	}
+
+	if config.EigenDA.Enable && config.DataAvailability.Enable && !config.BatchPoster.EnableEigenDAFailover {
+		return nil, errors.New("eigenDA and anytrust cannot both be enabled without EnableEigenDAFailover=true in batch poster config")
+	}
+
+	if config.EigenDA.Enable { // anytrust is enabled as an EigenDA failover
```

Comment:
how can I know that anytrust is enabled here? if just seems to check that eigenda is enabled